README.md

```diff
@@ -42,7 +42,8 @@ csrfToken.setToken('updated-csrf-token');
 To be able to get a new token if the current one expired you need to pass a retry object for the config with two keys:
 
 * `url`: URL to request the new token
-* `parseResponse`: A function that takes the request response as a parameter and return the new token
+* `parseResponse`: A function that takes the token request response as a parameter and return the new token
+* `isCSRFFailure` (optional): A function that takes the failed request response as a parameter and return true if it's a CSRF failure
```

Comment:
Could you provide an example? If I would need to use this function I'd have no idea how to get the status code or response body